apps/console/src/protected-app.tsx

```diff
@@ -602,41 +602,50 @@ export const ProtectedApp: FunctionComponent<AppPropsInterface> = (): ReactEleme
         }
         
         const resolveHiddenRoutes = (): string[] => {
-            const commonHiddenRoutes: string[] = [ 
-                ...AppUtils.getHiddenRoutes(), 
-                ...AppConstants.ORGANIZATION_ONLY_ROUTES 
+            const commonHiddenRoutes: string[] = [
+                ...AppUtils.getHiddenRoutes(),
+                ...AppConstants.ORGANIZATION_ONLY_ROUTES
             ];
             
-            const additionalRoutes: string[] = isOrganizationManagementEnabled ? (
-                (OrganizationUtils.isCurrentOrganizationRoot() 
-                && AppConstants.getSuperTenant() === tenant) || isFirstLevelOrg
-                    ? isPrivilegedUser
-                        ? loggedUserName === isSuperAdmin
-                            ? [ 
-                                ...commonHiddenRoutes, 
-                                ...AppConstants.ORGANIZATION_ROUTES 
-                            ]
-                            : [ 
-                                ...commonHiddenRoutes, 
-                                ...AppConstants.ORGANIZATION_ROUTES, 
-                                ...AppConstants.SUPER_ADMIN_ONLY_ROUTES 
-                            ]
-                        : loggedUserName === isSuperAdmin
-                            ? commonHiddenRoutes
-                            : [
-                                ...commonHiddenRoutes, 
+            function getAdditionalRoutes() {
```

Comment:
Shall we use arrow function here to be consistent?

Reply:
Since this is an inner function, shall we keep this as it is?